docs/_feature-guide-template.md

```diff
@@ -0,0 +1,83 @@
+import FeatureAvailability from '@site/src/components/FeatureAvailability';
+
+# About DataHub [Feature Name]
+
+<!-- All Feature Guides should begin with `About DataHub ` to improve SEO -->
+
+<!-- 
+Update feature availability; by default, feature availabilty is Self-Hosted and Managed DataHub
+
+Add in `saasOnly` for Managed DataHub-only features
+ -->
+
+<FeatureAvailability/>
+
+<!-- This section should provide a plain-language overview of feature. Consider the following:
+
+* What does this feature do? Why is it useful?
+* What are the typical use cases?
+* Who are the typical users?
+* In which DataHub Version did this become available? -->
+
+## [Feature Name] Setup, Prerequisites, and Permissions
+
+<!-- This section should provide plain-language instructions on how to configure the feature:
+
+* What special configuration is required, if any?
+* How can you confirm you configured it correctly? What is the expected behavior?
+* What access levels/permissions are required within DataHub? -->
+
+## Using [Feature Name]
+
+<!-- Plain-language instructions of how to use the feature
+
+Provide a step-by-step guide to use feature, including relevant screenshots and/or GIFs
+
+* Where/how do you access it?
+* What best practices exist?
+* What are common code snippets?
+ -->
+
+## Additional Resources
+
+<!-- Comment out any irrelevant or empty sections -->
+
+### YouTube
```

Comment:
Do we need a section on youtube directly? Maybe we can just say "Videos" or "Walkthroughs" or "Demos" or soemthing

Reply:
👍 good call - will change it to Videos